s3-commons/src/test/java/io/aiven/kafka/connect/tools/AwsCredentialBaseConfig.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.tools;
+
+import java.util.Map;
+
+import org.apache.kafka.common.config.ConfigDef;
+
+import io.aiven.kafka.connect.common.config.AivenCommonConfig;
+import io.aiven.kafka.connect.common.config.CompressionType;
+import io.aiven.kafka.connect.common.config.OutputFieldType;
+import io.aiven.kafka.connect.s3.S3SinkBaseConfig;
+
+public class AwsCredentialBaseConfig extends S3SinkBaseConfig {
+    public AwsCredentialBaseConfig(final Map<String, String> properties) {
+        super(configDef(), properties);
+    }
+
+    public static ConfigDef configDef() { // NOPMD UnusedAssignment
+        final ConfigDef configDef = getBaseConfigDefinition();
+        addS3RetryPolicies(configDef);
+        addAwsConfigGroup(configDef);
+        addAwsStsConfigGroup(configDef);
+        addDeprecatedConfiguration(configDef);
+        return configDef;
+    }
+
+    private static ConfigDef getBaseConfigDefinition() {
+        final ConfigDef definition = new ConfigDef();
+        addOutputFieldsFormatConfigGroup(definition, OutputFieldType.VALUE);
+        definition.define(AivenCommonConfig.FILE_NAME_TEMPLATE_CONFIG, ConfigDef.Type.STRING, null,
```

Comment:
As we know that AivenCommonConfig is deprecated, lets move these definitions to CommonConfig., also for the 2 below config.s

Reply:
These are just setup for tests.

Reply:
To elaborate and add context, the AWSCredentialProviderFactory uses this for running tests with some standard configuration added for those tests.